### DISCUSSION TITLE
Windows Instructions

### DISCUSSION THREAD

**Discussion by jzmoolman** [eugene-tarassov/vivado-risc-v#214] [General]
I am a Ph.D. Student at CU Boulder, Co, working on microarchitecture design for memory encryption unit for the risc-v.  We started with proprietary software but recently moved to Chislel/RocketChip.  I am following the Windows instructions, but they seem out of date.  Do you rather suggest that I use Ubuntu?  (PS.  I  have already installed the Vivado on Windows)

**Comment by eugene-tarassov** [comment]
> I am following the Windows instructions, but they seem out of date.

I'm not sure what you mean. The instructions look OK to me.

> Do you rather suggest that I use Ubuntu?

You have to use Ubuntu 20 LTS - it is the only supported OS.
You can either install Ubuntu on a PC, or use Windows WSL, or use Docker to run Ubuntu. Using dedicated PC is the easiest.
In any case, it has to be Ubuntu 20.04 LTS, exactly this version.

**Reply by jzmoolman** [reply]
Thanks for the quick feedback. I managed to build it and pull it into Vivado. The problem was with the Java version I was running.
